readme.md

```diff
@@ -24,7 +24,7 @@ Multi-platform and multi-language.
 - Bot behavior is customizable via configuration files (config as code)
```

Comment:
This would be a `docs:` type Semantic Commit. While I'm at it, you should also shorten the PR title a bit:

```diff
- fix(doc): update statement about replacements as they are now available for all managers
+ doc: replacement PRs are now available for more managers
```